sqlglot/dialects/oracle.py

```diff
@@ -328,3 +364,22 @@ def queryoption_sql(self, expression: exp.QueryOption) -> str:
         def coalesce_sql(self, expression: exp.Coalesce) -> str:
             func_name = "NVL" if expression.args.get("is_nvl") else "COALESCE"
             return rename_func(func_name)(self, expression)
+
+        def multitable_inserts_sql(self, expression: exp.MultitableInserts) -> str:
+            kind = self.sql(expression, "kind").upper()
+            inserts = []
+            for conditional_insert in t.cast(t.List[exp.ConditionalInsert], expression.expressions):
```

Comment:
Why do we need a `t.cast` here?

Reply:
We don't, `t.cast` just makes IDE show `expression.expressions` as a `List[exp.ConditionalInsert]`. I don't think it has any real effect.

Reply:
Let's remove it then please.